app/src/main/scala/org/alephium/explorer/config/ExplorerConfig.scala

```diff
@@ -34,12 +36,43 @@ import sttp.model.Uri
 import org.alephium.api.model.ApiKey
 import org.alephium.conf._
 import org.alephium.explorer.error.ExplorerError._
+import org.alephium.explorer.util.FileUtil
 import org.alephium.protocol.model.NetworkId
 import org.alephium.util
 
 @SuppressWarnings(Array("org.wartremover.warts.TryPartial"))
 object ExplorerConfig {
 
+  def getConfigFile(rootPath: Path, name: String): File =
+    rootPath.resolve(s"$name.conf").toFile
+
+  def getUserConfig(rootPath: Path): File = {
+    val file = getConfigFile(rootPath, "user")
+    FileUtil.createFileIfNotExists(file)
+    file
+  }
+
+  def loadConfig(rootPath: Path): Try[Config] = {
+    for {
+      userConfig <- parseConfigFile(getUserConfig(rootPath))
+    } yield {
+      val defaultConfig = ConfigFactory.parseResources("application.conf")
+      ConfigFactory.load(userConfig.withFallback(defaultConfig))
+    }
+  }
+
+  def parseConfigFile(file: File): Try[Config] =
+    try {
+      if (file.exists()) {
+        Success(ConfigFactory.parseFile(file))
```

Comment:
this will throw an error and catched by the `try`.

The same as in [the node ](https://github.com/alephium/alephium/blob/355b4dc5a1fdfaa2cb2d1c43e44dd430a4ab99c9/flow/src/main/scala/org/alephium/flow/setting/Configs.scala#L87-L97)